pegasus/sites.v3/code.org/public/ai/pl/101/index.haml

```diff
@@ -28,6 +28,8 @@ theme: responsive_full_width
       =hoc_s(:ai_pl_callout_desc_01)
     = view :ai_101_videos
 
+= view :dance_party_ai_skinny_banner
```

Comment:
Added the new banner here:

![ai_pl_101](https://github.com/code-dot-org/code-dot-org/assets/9256643/4051a042-86ca-4cc8-b6b3-b4ab8a6047ac)